### ISSUE TITLE
inst/tests do not run outside package

### ISSUE THREAD

**Issue by yrosseel** [simsem/simsem#65]
Perhaps this is only in my setup, but when testing packages (that depend on lavaan), I copy the examples and the tests (in inst/test) to a temp directory, and run the examples/tests over there (saving all results). This works well, except for simsem. The
reason is that many of the *.R files in inst/test/ rely on sourcing files with a relative path:
```r
source("../../R/model.R")
source("../../R/AllClass.R")
source("../../R/bind.R")
```
One solution could be to keep a local copy of these files in the inst/tests directory. Or to export more functions/classes, so that the tests only rely on library(simsem) to run. Or to remove tests that rely on private (non-exported) functions/classes.

**Comment by TDJorgensen** [comment]
These were leftovers from Sunthud that I never even opened. Many of the tests had to be deleted because there were obsolete (e.g., called functions that no longer existed after "simsemClassic"). 

The files now replace `source()` with `library(simsem)` and `library(testthat)`, which I suppose should work fine when you copy them for testing the influence of lavaan changes.  

```
devtools::install_github("simsem/simsem/simsem") # >= 0.5-15.901
```

**Comment by yrosseel** [comment]
Thanks. Most tests now work fine. I noticed two further issues:

1) in test_global.R, line 80 fails with:
```r
Error in buildModel(paramSet, modelType) : 
  The factor covariance (PS) matrix must be symmetric.
```
2) in test_missing.R, line 34 fails, as the percentage missing is 0.0947785

**Comment by TDJorgensen** [comment]
fixed in 0.5-15.902
Let me know if there are any other problems
